packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1065,13 +1039,6 @@ export class GarbageCollector implements IGarbageCollector {
 	 * before runnint GC next.
 	 */
 	private triggerAutoRecovery(nodePath: string) {
-		if (
-			!this.configs.tombstoneAutorecoveryEnabled ||
-			this.mc.config.getBoolean(disableAutoRecoveryKey) === true
-		) {
-			return;
```

Comment:
Hmmm won't this break v1/v2 collab compat?  If a Tombstone is loaded, it will send the GC op!

Reply:
Maybe we are missing test coverage here after all.  Or maybe I'm missing something else from this PR.

Reply:
yes, this will break v1/v2 collab. Let me see how to fix it.